piker/brokers/binance.py

```diff
@@ -141,16 +159,56 @@ def __init__(self) -> None:
         self._sesh.base_location = _url
         self._pairs: dict[str, Any] = {}
 
+        conf = get_config()
+        self.api_key = conf.get('api', {}).get('key')
+        self.api_secret = conf.get('api', {}).get('secret')
+
+        if self.api_key:
+            self._sesh.headers.update({'X-MBX-APIKEY': self.api_key})
+
+    def _get_signature(self, data: OrderedDict) -> str:
+        if not self.api_secret:
```

Comment:
Should probably add some more comments liking to docs about the signing of requests